adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -390,10 +394,18 @@ case class FeatureRDD(rdd: RDD[Feature],
   def saveAsGff3(fileName: String,
                  asSingleFile: Boolean = false,
                  disableFastConcat: Boolean = false) = {
+    val optHeaderPath = if (asSingleFile) {
+      val headerPath = "%s_head".format(fileName)
+      GFFHeaderWriter(headerPath, rdd.context)
```

Comment:
`GFFHeaderWriter` should be `GFF3HeaderWriter` to match `GFF3InFormatter` and elsewhere.  The GFF2/GTF specifications do not require a version pragma in the header.